fix: unsigned plugin message (#581)
salesforcecli/plugin-trust: messages/verify.md

- <%= config.bin %> <%= command.id %> --npm @scope/npmName

# flags.npm.summary

Specify the npm name. This can include a tag/version.

# flags.registry.summary

The registry name. The behavior is the same as npm.

# FailedDigitalSignatureVerification

A digital signature is specified for this plugin but it didn't verify against the certificate.

# InstallConfirmation

> This plugin is not signed by Salesforce. Only install the plugin if you trust its creator. Continue installation?,

Reviewer: ```suggestion
Salesforce hasn't signed this plugin. Install the plugin only if you trust its creator. Do you want to continue the installation?
```

Author: used the option from your note
